content/docs/command-reference/add.md

```diff
@@ -336,91 +339,78 @@ $ tree .dvc/cache
-[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-that could handle it.
+Sometimes you may want to add a large dataset currently found in an external
+location, so it becomes local to the project. But what if there's not enough
+disk space to download the data?
 
-The `--out` option lets you add external paths in a way that they are
+The `--out` option lets you add external data in a way that it's
 <abbr>cached</abbr> first, and then
 [linked](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
-to a given path inside the <abbr>workspace<abbr>. Let's initialize an example
-DVC project to try this:
-
-```dvc
-$ mkdir example # workspace
-$ cd example
-$ git init
-$ dvc init
-```
+to a given path inside the <abbr>workspace</abbr>. Combined with an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+and the right
+[kind of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache),
+this let's you avoid using the local file system completely.
 
-Now we can add a `data.xml` file via HTTP for example, putting it a local path
-in our project:
+For example, we can add a `data.xml` file via HTTP, outputting it to a local
+path in our project:
 
 ```dvc
-$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml
+$ dvc add https://data.dvc.org/get-started/data.xml -o raw/data.xml
 
 $ ls
 data.xml data.xml.dvc
 ```
 
-The resulting `.dvc` file will save the provided local `path` as if the data was
-already in the workspace, while the `md5` hash points to the copy of the data
-that has now been transferred to the <abbr>cache</abbr>. Let's check the
-contents of `data.xml.dvc` in this case:
+The local `data.xml` should be a symlink or hard link to the externally
+<abbr>cached</abbr> data copy. The resulting `.dvc` file will save the local
+`path` as if the data was already there before this command. Let's check the
+contents of `data.xml.dvc`:
 
 ```yaml
 outs:
   - md5: a304afb96060aad90176268345e10355
     nfiles: 1
-    path: data.xml
+    path: raw/data.xml
 ```
 
 > For a similar operation that actually keeps a connection to the data source,
 > please see `dvc import-url`.
 
-## Example: Transfer to remote storage
+## Example: Storing large data remotely {#straight-to-remote}
 
-When you have a large dataset in an external location, you may want to track it
-as if it was in your project, but without downloading it locally (for now). The
-`--to-remote` option lets you do so, while storing a copy
-[remotely](/doc/command-reference/remote) so it can be
-[pulled](/doc/command-reference/plots) later. Let's initialize a DVC project,
-and setup a remote:
+Similarly to the previous scenario, you may sometimes want to add a large
+dataset found externally into a regular <abbr>project</abbr> (with a local
+<abbr>cache</abbr>). Can it be done without downloading the data locally (for
+now)? Yes!
```

Comment:
`(for now)` - not sure I understand this 🤔

Reply:
Removed `(for now)`.

It was a reference to the fact that later you can `pull` it. But that's explained later in the example so no need for the obscure parenthesis 👍